### ISSUE TITLE
Mysql Install

### ISSUE THREAD

**Issue by coxced** [blueseerERP/blueseer#96]
I have work past all the errors but this one, Got any ideas!
I've done this 

show global variables like 'local_infile';
set global local_infile = 'ON';

"NOTE!!!:  This script should only be executed ONCE.   This script will blow away the previous database and data and install a clean schema with default data."
Previous Data will be lost...are you sure you want to install database? (enter: yes or no):yes
Enter the database server IP addr (ex: 10.15.2.2 or localhost):localhost
Enter the administrator password for the MySQL Database:********
Enter the two character language code (en,de,fr,es,tr,etc):en
"creating bs.cfg file...."
"creating temp my.cnf..."
"creating database schema...."
"loading user account...."
"loading 'some' default records for tables from language pack en....."
_**ERROR 3948 (42000) at line 1 in file: 'sq_mysql.txt': Loading local data is disabled; this must be enabled on both the client and server sides**_
"finished install..."
"Login = 'admin' and Password = 'admin'"
"Click the login.bat file in the BlueSeer Directory to log in"

I'm using windows 10

Thanks, Cedric

**Comment by vaughnte** [comment]
You're real close....the answer is up above in your comment.   The error is steming from the security feature of MySQL that prevents loading local data.    Open a mysql prompt...and type:

show global variables like 'local_infile';

if it is off or blank...then type:

set global local_infile = 'ON';

then run the show global variables like 'local_infile' again...you should see it turned on.   
Then...rerun the script.   It should move beyond the ERROR 3948.

**Comment by vaughnte** [comment]
Note...I've also seen solutions of :
set global local_infile = 1;
set global local_infile = True;
...but in the end when you run "show global variables like 'local_infile'...it should be 'ON'.   It should then allow you to run it.

**Comment by coxced** [comment]
Ok now ive got this error 1366 (HY000)

"loading 'some' default records for tables from language pack en....."
ERROR 1366 (HY000) at line 12 in file: 'sq_mysql.txt': Incorrect integer value: '' for column 'edd_eledelim' at row 1
"finished install..."
"Login = 'admin' and Password = 'admin'"
"Click the login.bat file in the BlueSeer Directory to log in"

**Comment by vaughnte** [comment]
yeah...that looks like a bug I just introduced with a recent upload.    I've updated the data file in the project source with a proper integer value.    For a quick fix....Do the following:
download sf/data/en/edidoc.csv from the source repository and overwrite the same file in your install directory at: blueseer/data/en/edidoc.csv  ....then rerun the script.   
It will complete after that.   I'll upload a fresh install package later today if you want to take that route.
